.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2 # use CircleCI 2.0
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+    environment:
+      CC_TEST_REPORTER_ID: ${TEST_REPORTER_ID}
+    working_directory: ~/authors-haven # directory where steps will run
+    docker:
+      - image: circleci/node:10.3.0
+    steps: # a collection of executable commands
+      - checkout # special step to check out source code to working directory
+      - restore_cache: # special step to restore the dependency cache
+          # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm-authors-haven-dependencies
+          command: npm install
+      - save_cache: # special step to save the dependency cache
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run: # run tests
+          name: test
+          command: npm run test:coverage
+      - run:
+          name: Install JUnit coverage reporter
+          command: npm install jest-junit
+      - run:
+          name: Run tests with JUnit as reporter
+          command: npm run jest:coverage
+          environment:
+            JEST_JUNIT_OUTPUT: "reports/junit/js-test-results.xml"
+      - store_artifacts: # special step to save test results as as artifact
+          # Upload test summary for display in Artifacts: https://circleci.com/docs/2.0/artifacts/
+          path: reports/junit
+          prefix: tests
+      - store_test_results: # for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
+          path: reports/junit
+      # See https://circleci.com/docs/2.0/deployment-integrations/ for deploy examples
```

Comment:
new line

Reply:
![image](https://user-images.githubusercontent.com/32358797/60668149-2c147580-9e63-11e9-9810-d678f2c001ad.png)

Circleci/config.yml would throw an error if I create a new line as shown in the screenshot above.